.drone.yml

```diff
@@ -95,6 +96,84 @@ trigger:
     - develop
 
 ##############################################################################################################
+# This pipeline test the launcher and its rpc sercver
+##############################################################################################################
+---
+kind: pipeline
+name: launcher
+
+clone:
+  disable: true
+
+environment:
+  SODIUM_USE_PKG_CONFIG: 1
+  NETWORK: sandbox
+  TEZEDGE_NODE_RPC_CONTEXT_ROOT: http://tezedge-node-sandbox-run:18732
+  TEZEDGE_LAUNCHER_RPC_CONTEXT_ROOT: http://tezedge-launcher-sandbox-run:3030
+
+steps:
+  - name: tezedge-launcher-run
+    image: simplestakingcom/tezedge-ci-builder:latest
+    pull: if-not-exists
+    user: root
+    detach: true
+    volumes:
+      - name: build
+        path: /artifacts
+      - name: sandbox-data
+        path: /tmp/sandbox
+    environment:
+      SODIUM_USE_PKG_CONFIG: 1
+      RUST_BACKTRACE: 1
+    commands:
+      - cp /artifacts/protocol-runner /tmp/sandbox/
+      - rust_libs=$(echo "`rustup show home`/toolchains/`rustup show active-toolchain | tr " " "\n" | head -1`/lib")
+      - export LD_LIBRARY_PATH="/artifacts/ffi:$rust_libs"
+      - echo "LD_LIBRARY_PATH - $LD_LIBRARY_PATH"
+      - /artifacts/launcher --launcher-rpc-port 3030 --light-node-path /artifacts/light-node --logl-level info
+
+- name: start-tezedge-node-via-rpc
+  user: root
+  image: cfmanteiga/alpine-bash-curl-jq
+  volumes:
+    - name: sandbox-data
+      path: /tmp/sandbox
+  commands:
+    - sleep 5
+    - LAUNCHER_URL=$${TEZEDGE_LAUNCHER_RPC_CONTEXT_ROOT}
+    - resp=$(curl --request POST "$LAUNCHER_URL/start --data-raw '{"config_file":"./light_node/etc/tezedge/tezedge.config","identity_expected_pow":0,"disable_bootstrap_lookup":true,"network":"sandbox","peer_thresh_low":1,"peer_thresh_high":1,"sandbox_patch_context_json_file":"./light_node/etc/tezedge_sandbox/sandbox-patch-context.json","protocol_runner":"./target/release/protocol-runner"}' " | jq -S .)
+    - sleep 5
+  
+- name: stop-tezedge-node-via-rpc
```

Comment:
@adonagy 
to verify startup, you can use:

```
- name: wait-for-tezedge-to-start
  image: simplestakingcom/tezos-node-bootstrap:latest
  commands:
    - tezos-node-bootstrap --bootstrap --level=0 --node_tezedge=$${TEZEDGE_NODE_RPC_CONTEXT_ROOT}
```

Reply:
Yeah, good idea.